src/search/components/RealTimeKeyword/RealTimeKeyword.tsx

```diff
@@ -89,18 +68,16 @@ export const RealTimeKeyword = () => {
             <br />
             검색한 키워드
           </StyledHeaderTitle>
-          {realTime ? (
-            <StyledHeaderTime>
-              {`${realTime.year}.${realTime.month}.${realTime.day} ${realTime.hour}:${realTime.min} 기준`}
-            </StyledHeaderTime>
-          ) : null}
+          {isLoading && <StyledHeaderTime>연결 중입니다</StyledHeaderTime>}
```

Comment:
```suggestion
         <Suspense fallback={<StyledHeaderTime>연결 중입니다</StyledHeaderTime>}>
           <StyledHeaderTime>{`${realTime} 기준`}</StyledHeaderTime>
         </Suspense>
```

다만, 저희가 v5 사용중이라서 query 자체를 suspenseQuery로 바꿔야 한다는 점... 점점점...

Reply:
문서를 참고하지 않고 쓰면 이렇게 되는군요.. 관련 문서 참고 후 수정해두도록 하겠습니다! 감사합니다!